### PR TITLE
should fix #101

### LINKED ISSUE #101: Twitter Integration Issues
Hey @dvargas92495 , was testing out the twitter integration on master, and running into this error - 
`{"message":"Missing Authentication Token"}`

How do we go about debugging this? Console logs?

### PR DESCRIPTION
**Pull request by makalaaneesh**

### CODE REVIEW

**Comment by makalaaneesh** [comment]
https://developer.twitter.com/en/docs/authentication/api-reference/token

**Review by dvargas92495 — COMMENTED** [review]
one possible way to test the aws lambda locally is to run `npm run build:lambda` locally. It'll output a file that you could run with `node ./out/twitter-search_get.js`. You'll prob want to add a `handler().then(r => console.log(r.data))` at the bottom of the file before building so that it actually calls the method

File: src/lambda-helpers.ts [hunk 1]
```diff
@@ -47,17 +47,20 @@ export const getGithubOpts = () => ({
 const twitterConsumerKey = process.env.TWITTER_CONSUMER_KEY || "";
 const twitterConsumerSecret = process.env.TWITTER_CONSUMER_SECRET || "";
 
-export const getTwitterOpts = async () => {
-  const twitterBearerTokenResponse = await wrapAxios(
```
[the remark this answers is not in the document]
Comment (makalaaneesh) [reply src/lambda-helpers.ts:51]: done.

**Comment by makalaaneesh** [comment]
> one possible way to test the aws lambda locally is to run `npm run build:lambda` locally. It'll output a file that you could run with `node ./out/twitter-search_get.js`. You'll prob want to add a `handler().then(r => console.log(r.data))` at the bottom of the file before building so that it actually calls the method

Thanks for this! This was very useful! Made the fixes, and tested the lambda locally!

Any way we can test end-to-end locally? i.e. testing the button listener directly and receiving response from lambda? Should I be calling `node build/twitter.js`?

**Review by dvargas92495 — APPROVED** [review]
currently the lambda code needs to be merged to master in order to be able to test :/. So... doing that now!

**Pull request merged**